at_commons/lib/src/verb/lookup_verb_builder.dart

```diff
@@ -26,15 +26,20 @@ class LookupVerbBuilder implements VerbBuilder {
 
   String? operation;
 
+  // if set to true, returns the value of key on the remote server instead of the cached copy
+  bool byPassCache = false;
+
   @override
   String buildCommand() {
-    String command;
+    String command = 'lookup:';
+    if (byPassCache == true) {
+      command += 'bypassCache:$byPassCache:';
```

Comment:
Do we have to deal with a new client talking to an old server that does not understand bypassCache? If yes, we have to consider using a flag to indicate that server supports bypass cache.

Reply:
+1 to Jagan's comment, or, we do not merge this change on the client side until the required version of the server has been rolled out to all of prod

Reply:
@murali-shris do you have a corresponding pull request for the changes to the server verb handlers? Also, should we do an initial pull request for at_commons with just the new at_constants.dart and syntax.dart? (so they are available to the server)

Reply:
@VJag @gkc This is the plan for roll out
1. merge and publish at_commons. Even if any client uses latest version of at_commons there won't be a breaking change since byPassCache will be set only if client sets it
2. merge and release server to canary and prod
3. merge and publish at_client

Reply:
@gkc PR for server changes
https://github.com/atsign-foundation/at_server/pull/677/files
EndtoEnd test changes require some more work to be done in order to bring down and bring up server as a part of test steps.
Below comment from Chris
OK... two things from my perspective:
1. The branch needs to become a PR so it can be reviewed properly
2. The TODOs need to find their way out of the test code and into the scripts we use to run the tests in the at_server.yaml workflow.
e.g. at the moment we call a script to update the image we have for the atsigns:
      # Logs into CICD VMs and runs script to update to just pushed image
      - name: update image on cicd VMs
        uses: appleboy/ssh-action@v0.1.4
        with:
          host: "cicd1.atsign.wtf,cicd2.atsign.wtf"
          username: ubuntu
          key: ${{ secrets.CICD_SSH_KEY }}
          script: |
            ./update_image.sh
We're going to have to do something similar to reach onto the VMs and stop/start the atsign secondaries.
It might help to have @sitaram kalluri involved, as he's spent a lot of time on this workflow previously.
I'll drop the update_image.sh script into the automation directory so it can be seen there.

Reply:
Which test case(s) require(s) that we restart the server?

Reply:
at_end2end_test/test/cached_key_test.dart